### PR TITLE
chore(query/influxql): remove the influxql transpiler

### PR DESCRIPTION
**Pull request by jsternberg**

### CODE REVIEW

**Review by lesam — APPROVED** [review]

File: http/query.go [hunk 1]
```diff
@@ -93,14 +86,10 @@ func (r QueryRequest) Validate() error {
 		return errors.New(`request body requires either query or AST`)
 	}
 
-	if r.Type != "flux" && r.Type != "influxql" {
```
Comment (lesam) [inline http/query.go:96]: @jsternberg This needs a docs change right?

Comment (jsternberg) [reply http/query.go:96]: Is that the openapi specification?

Comment (jsternberg) [reply http/query.go:96]: It looks like the openapi specification has had the influxql portions removed either always or for some time. It's not in there so this isn't listed as a valid option according to our api contract.

File: http/api_handler.go [hunk 2]
```diff
@@ -214,11 +213,11 @@ func NewAPIHandler(b *APIBackend, opts ...APIHandlerOptFn) *APIHandler {
 	writeBackend := NewWriteBackend(b.Logger.With(zap.String("handler", "write")), b)
 	h.Mount(prefixWrite, NewWriteHandler(b.Logger, writeBackend,
 		WithMaxBatchSizeBytes(b.MaxBatchSizeBytes),
-		//WithParserOptions(
+		// WithParserOptions(
 		//	models.WithParserMaxBytes(b.WriteParserMaxBytes),
 		//	models.WithParserMaxLines(b.WriteParserMaxLines),
 		//	models.WithParserMaxValues(b.WriteParserMaxValues),
-		//),
+		// ),
```
Comment (stuartcarnie) [inline http/api_handler.go:220]: I would say this is worth removing, rather than reformatting

**Review by stuartcarnie — APPROVED** [review]
Very satisfying! 🧹 → 🗑️ 

I only had one take-it or leave-it comment, otherwise, great stuff!

**Comment by lesam** [comment]
Rebased with minimal changes

**Pull request merged**